docker-compose.yml

```diff
@@ -2,26 +2,30 @@ notaryserver:
   build: .
   dockerfile: server.Dockerfile
   links:
-   - notarymysql
-   - notarysigner
-  ports:
-   - "8080"
-   - "4443:4443"
+    - notarymysql
+    - notarysigner
   environment:
-   - SERVICE_NAME=notary
-  command: -config=fixtures/server-config.json
+    - SERVICE_NAME=notaryserver
+  ports:
+    - "8080"
+    - "4443:4443"
+  entrypoint: /bin/bash
+  command: -c "./migrations/migrate.sh; notary-server -config=fixtures/server-config.json"
 notarysigner:
-  volumes:
-   - /dev/bus/usb/003/010:/dev/bus/usb/002/010
-   - /var/run/pcscd/pcscd.comm:/var/run/pcscd/pcscd.comm
   build: .
   dockerfile: signer.Dockerfile
   links:
-   - notarymysql
-  command: -config=fixtures/signer-config.json
+    - notarymysql
+  environment:
+    - SERVICE_NAME=notarysigner
+  entrypoint: /bin/bash
+  command: -c "./migrations/migrate.sh; notary-signer -config=fixtures/signer-config.json"
 notarymysql:
   volumes:
-    - notarymysql:/var/lib/mysql
-  build: ./notarymysql/
+    - ./notarymysql/docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d
+    - ./notarymysql/data:/var/lib/mysql
+  image: mariadb:10.1.10
   ports:
     - "3306:3306"
+  environment:
+      MYSQL_ALLOW_EMPTY_PASSWORD: true
```

Comment:
I'm testing this on my machine and getting this warning:

```
WARNING: There is a boolean value in the 'environment' key.
Environment variables can only be strings.
Please add quotes to any boolean values to make them string (eg, 'True', 'yes', 'N').
This warning will become an error in a future release.
```

Could we use a string value instead?